src/server/utils/bucket_diff.js

```diff
@@ -69,38 +69,47 @@ class BucketDiff {
             //     ]
             // }
             keys_diff_map: {},
+            keys_del_map: {},
```

Comment:
Need to add a comment on the stracture of `keys_del_map` same as the comment for `keys_diff_map` 
amd maybe change the name of `keys_del_map` to `keys_del_diff_map`

Reply:
Done, Thanks.